core/src/Kokkos_BitManipulation.hpp

```diff
@@ -398,7 +404,7 @@ KOKKOS_FUNCTION std::enable_if_t<sizeof(To) == sizeof(From) &&
                                      std::is_trivially_copyable_v<From>,
                                  To>
 bit_cast_builtin(From const& from) noexcept {
-  return bit_cast<To>(from);  // no benefit to call the _builtin variant
+  return Kokkos::bit_cast<To>(from);  // no benefit to call the _builtin variant
```

Comment:
Please propose that change on its own because that one is a no brainer.
Add a comment that it is qualified because we don't want ADL to kick in.

Reply:
See #6122.